pkg/scaffold/internal/util/path.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"strings"
+
+	"github.com/gobuffalo/flect"
+	"sigs.k8s.io/kubebuilder/pkg/fmt"
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/resource"
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/webhook"
+)
+
+func Path(path string, parameters ...interface{}) string {
```

Comment:
IHMO: -1 for this change. In POV it makes the manutence ability be lower and do not improve the performance and/or design either.

Reply:
@camilamacedo86 Please consider reading the previous comment, especifically the last paragraph, where I describe the functionality it adds. Should write it in the description too though.